session/src/test/java/uk/gov/di/ipv/cri/common/api/handler/SessionHandlerTest.java

```diff
@@ -39,6 +39,7 @@
 import static uk.gov.di.ipv.cri.common.api.handler.SessionHandler.SESSION_ID;
 import static uk.gov.di.ipv.cri.common.api.handler.SessionHandler.STATE;
 
+@SuppressWarnings("rawtypes")
```

Comment:
Instead of suppressing the warnings use ( this can be seen in AccessTokenHandlerTest ):
```
Map<String, Object> responseBody =
                new ObjectMapper().readValue(responseEvent.getBody(), new TypeReference<>() {});
```
where the warnings appear